includes/countries.php

```diff
@@ -255,4 +261,37 @@
 
 	asort($pmpro_countries);
 
-	$pmpro_countries = apply_filters("pmpro_countries", $pmpro_countries);
+	/**
+	 * Filter the list of countries.
+	 *
+	 * @param array $pmpro_countries The list of countries with their codes as keys and names as values.
+	 */
+	$pmpro_countries = apply_filters( 'pmpro_countries', $pmpro_countries );
```

Comment:
May be worth moving this before the return so the "select a country" is included.

Reply:
I thought about that, but what held me back is that then we'd probably want to pass `$include_empty` to the `pmpro_countries` filter as well so that custom code knows whether the empty option should be here.

I thought that this solution was ok to ensure that the empty option is always present and at the beginning of the list when needed, but I'd be open to making that swap

Reply:
That's okay with me leaving it as is. Excited for this one 👍